ui/components/TracePage/CallName.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from "classnames";
+
+export const CallName = ({ className, name, cls }: CallNameProps) => {
+  const displayName = (name === "execute" || name === "run") && cls ? cls : name;
+  return (
+    <div className="flex items-center gap-1">
+      {cls && cls !== displayName ? (
+        <span className={classNames(className, "text-gray-500")}>{getFormattedName(cls)}:</span>
+      ) : undefined}
+      <span className={className}>{getFormattedName(displayName)}</span>
+    </div>
+  );
+};
+
+export interface CallFunction {
+  name: string; // function name
+  cls?: string; // class name for methods
+}
+
+interface CallNameProps extends CallFunction {
```

Comment:
Let's inline this since it's used in only one place

Reply:
I'd rather keep it for the reuse of `CallFunction`.

Reply:
d'oh